Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -514,6 +516,39 @@ class BrowserViewController: UIViewController {
         }
     }
 
+    private func animateBookmarkStar() {
+        let duration = 0.5
+        var offset: CGFloat
+        var button: UIButton!
```

Comment:
Since these are guaranteed to have a value after the else clause, you can change these definitions to let instead of var.